skyportal/facility_apis/__init__.py

```diff
@@ -4,4 +4,5 @@
 from .sedmv2 import SEDMV2API
 from .lt import IOOAPI, IOIAPI, SPRATAPI
 from .lco import SINISTROAPI, SPECTRALAPI, FLOYDSAPI, MUSCATAPI
+from .swift import UVOTAPI
```

Comment:
Suggest changing this to SWIFTAPI or UVOT_XRTAPI to get the naming consistent with the instruments that will be requested.

Reply:
I changed it to UVOTXRTAPI.